test/behavior/array.zig

```diff
@@ -808,3 +808,11 @@ test "tuple initialized through reference to anonymous array init provides resul
     try expect(foo[0] == 12345);
     try expect(@intFromPtr(foo[1]) == 0x1000);
 }
+
+test "@bitSizeof on array of structs" {
+    const S = struct {
+        foo: u64,
+    };
+
+    try expectEqual(128, @bitSizeOf([2]S));
+}
```

Comment:
I'd probably put this in `test/behavior/sizeof_and_typeof.zig`. (also, capitalization issue on `@bitSizeof` in the test name)

Reply:
Moved the test 👍